google_takeout_parser/parse_json.py

```diff
@@ -81,7 +81,7 @@ def _parse_json_activity(p: Path) -> Iterator[Res[Activity]]:
                 time=parse_json_utc_date(time_str),
                 subtitles=subtitles,
                 details=[
-                    d["name"]
+                    d.get("name", "")
```

Comment:
should be reverted to direct access, unless theres some data you've found that is missing a name, the`""` empty string is just introducing new junk data